### DISCUSSION TITLE
Schedule

### DISCUSSION THREAD

**Discussion by sammou00** [hyf-Group2-fp/Just4Giving#16] [General]
Your working Schedule goes here ....

**Comment by sammou00** [comment]
Every day  10:30 AM to 8 PM

**Reply by bermarte** [reply]
Every day  10:30 AM - 8 PM

**Comment by DIVYASREE345** [comment]
Monday, Wednesday,Friday 10 AM - 9 PM
Tuesday & Thursday  1 PM - 5 PM

**Comment by kemmy72** [comment]
Sunday  10:30 AM - 8 PM
Monday 1 PM - 5 PM
Tuesday    1 PM - 9 PM
Wednesday  5 PM - 9 PM 
Thursday & Friday 5:30 PM - 8 PM 
Saturday    10:30 AM - 8 PM

**Comment by AnisyaPurnama** [comment]
Monday  To  Friday  6 PM - 8 PM
Sunday 11 AM - 8 PM

**Comment by Brainketunze** [comment]
Monday To Friday 11 AM - 8 PM
Sunday 11 AM - 7 PM

**Comment by sharafcs50** [comment]
Daily 7PM Onwards.
